.github/workflows/microshed-ci.yml

```diff
@@ -71,54 +82,56 @@ jobs:
               wildfly-app
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up Java
-      uses: joschi/setup-jdk@v2
-      with:
-        java-version: 11
-        openjdk_impl: openj9
-    - uses: testspace-com/setup-testspace@v1
-      with:
-        domain: ${{ github.repository_owner }}
-    - name: Apply repository caches
-      uses: actions/cache@v2
-      with:
-        path: |
-          ~/.gradle/caches/modules-2
-          !~/.gradle/caches/modules-2/org.microshed*
-          ~/.gradle/wrapper
-          ~/.m2/repository/
-          !~/.m2/repository/org/microshed/
-        key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}-${{ hashFiles('**/pom.xml') }}
-        restore-keys: |
-          ${{ runner.os }}-gradle-
-    - name: Run integration tests
-      shell: bash
-      env:
-        TEST_PROJECTS: ${{matrix.projects}}
-        CATEGORY: ${{matrix.category}}
-      run: |
-        chmod +x gradlew
-        echo "org.gradle.daemon=false" >> gradle.properties
-        
-        echo "Will be running projects $TEST_PROJECTS"
-        for TEST_PROJECT in $TEST_PROJECTS
-        do
-          if [[ ! -d "sample-apps/$TEST_PROJECT" ]]; then
-            echo "::error::Test project $TEST_PROJECT does not exist.";
-            exit 1;
-          fi
-        done
-        
-        for TEST_PROJECT in $TEST_PROJECTS
-        do
-          echo "### BEGIN running test project $TEST_PROJECT"
-          ./gradlew :microshed-testing-$TEST_PROJECT:test
-          echo "### END running test project $TEST_PROJECT";
-        done
-        
-        echo "Done running all tests."
-    - name: Upload test results
-      if: always()
-      run: |
-        testspace [${{matrix.category}}]sample-apps/*/build/test-results/**/*.xml
+      - uses: actions/checkout@v3
+      - name: Set up Java
+        uses: actions/setup-java@v3
+        with:
+          java-version: 11
+          distribution: 'zulu'
+      - name: Apply repository caches
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches/modules-2
+            !~/.gradle/caches/modules-2/org.microshed*
+            ~/.gradle/wrapper
+            ~/.m2/repository/
+            !~/.m2/repository/org/microshed/
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+      - name: Setup docker host
+        run: echo "DOCKER_HOST=unix:///run/user/$(id -u)/podman/podman.sock" >> "$GITHUB_ENV"
+      - name: Check podman
+        run: docker version
+      - name: Set up testcontainers for podman
+        run: |
+          echo ryuk.container.privileged=true > ~/.testcontainers.properties
+      - name: Login to Docker Hub
+        uses: redhat-actions/podman-login@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME  }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+          registry: docker.io
```

Comment:
I don't think these setup steps are necessary.  A docker daemon should already be running on the GitHub Actions machine.

Reply:
Without the podman setup: com.github.dockerjava.api.exception.NotFoundException: Status 404: {"message":"No such image: testcontainers/ryuk:0.3.0"} is the error...?